setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = release-bot
+version = attr: release_bot.version.__version__
```

Comment:
I have one more request to make: can we also add setuptools_scm and get rid of explicit versioning?

https://github.com/TomasTomecek/ansible-bender/blob/master/setup.cfg#L38
https://github.com/TomasTomecek/ansible-bender/blob/master/ansible_bender/__init__.py#L11

if you don't feel like it, don't do it and we can discuss once I return

actually it's probably a good idea not to overload this

Reply:
Yes, I was thinking about it and I think it'd be better to do it in separate PR. #141